specs/derivation.md

```diff
@@ -473,28 +473,31 @@ updated, such that the batch-sender authentication is always accurate to the exa
 
 ### L1 Retrieval
 
-In the *L1 Retrieval* stage, we read the block we get from the outer stage (L1 traversal), and extract data from it.
-By default, the rollup operates on calldata retrieved from [batcher transactions][g-batcher-transaction] in the block,
-for each transaction:
+In the *L1 Retrieval* stage, we read the block we get from the outer stage (L1 traversal), and
+extract data from its [batcher transactions][g-batcher-transaction]. A batcher
+transaction is one with the following properties:
 
-- The receiver must be the configured batcher inbox address.
-- The sender must match the batcher address loaded from the system config matching the L1 block of the data.
+- The receiver is the configured batcher inbox address.
```

Comment:
We could be more clear with the language here. While receiver makes sense, it does leave some room for ambiguity. Thoughts on using the same language in the execution spec with a hyperlink to it? The [spec](https://github.com/ethereum/execution-specs/blob/3fe6514f2d9d234e760d11af883a47c1263eff51/src/ethereum/frontier/fork_types.py#L52C31-L52C31) uses `to`. So something like "The `to` field is the configured batcher inbox address"

Reply:
I like referencing the "To:" field which I think is very clear & unambiguous.  changed.i

Reply:
Can you add the same hyperlink that i included above directly into the spec?

Reply:
done